### ISSUE TITLE
Integration with Microsoft Domains

### ISSUE THREAD

**Issue by mlaurent205** [freeipa/ansible-freeipa#638]
Is there a way to modify this parameter so that it appears "true"?
https://github.com/freeipa/ansible-freeipa/blob/b598470c2bba5e7cb0792074792e1aa5ff67f6fa/roles/ipaclient/action_plugins/ipaclient_get_otp.py#L105

**Comment by t-woerner** [comment]
There is no way at the moment to change this in the module. Is `dns_lookup_realm` always needed for AD integration?

**Comment by RobVerduijn** [comment]
I think it's needed to find the SRV records of the AD controller.

Rob

**Comment by abbra** [comment]
Nope. You need it only to be able to find the realm, not the KDCs.
What I would suggest @t-woerner to do is to add 'includedir /etc/krb5.conf.d/' at the top of the snippet instead. This will allow you to put the required modifications there and they'll override the snippet's parts, like we do on the actual `ipa-client-install`.

**Comment by mlaurent205** [comment]
I use these parameters to be able to authenticate my users with the freeipa servers. The authentication domain is different from the FreeIPA domain.

Currently, I make a manual change, after installing FreeIPA Client on the servers, to be able to use it. When would it be great if he could get the role.

**Comment by t-woerner** [comment]
This is the ipaclient_get_otp action plugin that is executed on the server to get an OTP for the client deployment. Is `dns_lookup_realm = true` needed for admin?

**Comment by abbra** [comment]
@t-woerner I think that would probably be needed if someone has configured AD admins to be able to enroll IPA clients (this is possible).
